package.json

```diff
@@ -6,9 +6,9 @@
     "clear": "rm package-lock.json && rm -rf node_modules",
     "prettier": "prettier \"src/**/!(static)/**/*.*(js|jsx)\"",
     "prettier:write": "prettier \"src/**/!(static)/**/*.*(js|jsx)\" --write",
-    "test": "jest",
-    "test:watch": "jest --watch",
```

Comment:
These two lines are used by the "test" job and probably why this build is failing on circle.